### PR TITLE
Fix SomaFM controller

### PR DESCRIPTION
**Pull request by ddbeck**
I also had the problem reported in #251 with SomaFM not responding to media keys. I've updated the corresponding controller to fix the problem (or, at least, it fixed it for me). Let me know if there's anything else I can do to get this PR into a mergeable state. Thanks!

### CODE REVIEW

File: code/js/controllers/SomaFMController.js [hunk 1]
```diff
@@ -5,7 +5,10 @@
 
   new BaseController({
     siteName: "SomaFM",
+    buttonSwitch: true,
     play: "#playBtn",
-    pause: "#stopBtn"
+    pause: "#stopBtn",
+
+    playState: "#stopBtn",
```
Comment (berrberr) [inline code/js/controllers/SomaFMController.js]: Remove trailing `,`

**Comment by berrberr** [comment]
Thanks @ddbeck, looks good to me after that one little change

**Comment by ddbeck** [comment]
Trailing comma removed. Thanks!

**Comment by berrberr** [comment]
Thanks!

**Comment by inverse** [comment]
Thnx for the fix!

**Pull request merged**